apps/federatedfilesharing/lib/OCM/CloudFederationProviderFiles.php

```diff
@@ -250,7 +250,8 @@ public function shareReceived(ICloudFederationShare $share) {
 				\OC::$server->getGroupManager(),
 				\OC::$server->getUserManager(),
 				$shareWith,
-				\OC::$server->query(IEventDispatcher::class)
+				\OC::$server->query(IEventDispatcher::class),
+				\OC::$server->getLogger()
```

Comment:
```suggestion
				\OC::$server->get(LoggerInterface::class)
```

Also need to add the use statement

Reply:
cool, thanks for the hints.
I wasn't aware that it's possible to pass an exception in the LoggerInterface